### PR TITLE
Added async support for flushall() and flushdb()

### PR DESCRIPTION
**Pull request by cpwr**
Also, docstring has been updated

### CODE REVIEW

**Comment by cpwr** [comment]
@popravich Алексей, сориентируйте, пожалуйста, как исправить сие недоразумение

File: aioredis/commands/server.py [hunk 1]
```diff
@@ -126,14 +126,26 @@ def debug_segfault(self, key):
         # won't test, this probably works
         return self.execute(b'DEBUG', 'SEGFAULT')  # pragma: no cover
 
-    def flushall(self):
-        """Remove all keys from all databases."""
-        fut = self.execute(b'FLUSHALL')
+    def flushall(self, async=False):
```
Comment (popravich) [inline aioredis/commands/server.py]: `async` is a reserved keyword, argument must be renamed (maybe `async_flush` or something better)
https://docs.python.org/3.7/reference/lexical_analysis.html#keywords

File: aioredis/commands/server.py [hunk 2]
```diff
@@ -126,14 +126,26 @@ def debug_segfault(self, key):
...
+        """
+        Remove all keys from all databases.
+        :param async: lets the entire dataset to be freed asynchronously. Defaults to False
```
Comment (popravich) [inline aioredis/commands/server.py]: Please keep line length less than 80 characters (`make flake` will show this error).
Also add a line break after first sentence to make a separate paragraph for parameter description.

**Comment by cpwr** [comment]
- Maybe I should rename parameter to `sync` (since it's not a keyword) and set it's default value to `True`?
- Should I write some new tests, or should I change existing, like `tests/generic_commands_test.py:500` to use new features?

**Comment by popravich** [comment]
That's ok, thanks.

**Pull request merged**